src/mono/wasm/runtime/loader/config.ts

```diff
@@ -224,6 +224,9 @@ export async function mono_wasm_load_config(module: DotnetModuleInternal): Promi
     if (!configFilePath) {
         normalizeConfig();
         loaderHelpers.afterConfigLoaded.promise_control.resolve(loaderHelpers.config);
+        if (!loaderHelpers.config.startupMemoryCache) {
```

Comment:
Importing and invoking library initializers should also be part of this branch. Do you want to include it in this PR or should I file it separately? It should end up in 8 as well

Reply:
done